listenbrainz/webserver/static/js/src/RecentListens.test.tsx

```diff
@@ -28,6 +28,7 @@ const {
   spotify,
   user,
   webSocketsServerUrl,
+  tryHarder,
```

Comment:
What does this do? Perhaps we can use a more descriptive name. Right now, I have no idea what try harder means.

Reply:
Updated to `searchLargerTimeRange`